lib/s3-html/log.html

```diff
@@ -589,24 +587,34 @@
     });
 
     // We also have CSS based filters for our tests
+    const selectorTestRetried = "c-filter-only-failed-retried"
     const testingToolbar = document.querySelector('#testing-toolbar')
-    const toggleGroupClass = "pf-v6-c-toggle-group__button";
     testingToolbar.innerHTML = Mustache.render(test_filter_toolbar);
-    Array.from(testingToolbar.getElementsByClassName(toggleGroupClass)).forEach(function (button) {
-        button.addEventListener(
-            "click",
-            () => {
-                button.classList.toggle("pf-m-selected")
-                const dataFilter = button.getAttribute("data-filter")
-                document.querySelectorAll(`.${toggleGroupClass}:not([data-filter="${dataFilter}"])`)
-                    .forEach( el => {
-                        document.body.classList.remove(el.getAttribute("data-filter"));
-                        el.classList.remove("pf-m-selected");
-                    });
-                document.body.classList.add(dataFilter);
-            }
-        )
-    });
+    const failedFilter = testingToolbar.querySelector("#switch-for-failed-filtering");
+
+    function handleFilterSelection(checkbox) {
+        // Apply the clicked filter and add it to localStorage
+        if (checkbox.checked) {
+            document.body.classList.add(selectorTestRetried);
+            localStorage.setItem(selectorTestRetried, checkbox.checked);
```

Comment:
This still rubs me the wrong way. This *really* should not be "configuration", this is more like temporary state. If you look at two different test logs, your choice is independent for both of them.

Hence, `sessionStorage` feels a bit less sticky and more appropriate here. TBH I think a fragment value would be even better.